src/silvistat/storage.py

```diff
@@ -1,68 +1,96 @@
 import pdal
 import tiledb
 import numpy as np
+from math import floor
 
 from . import Bounds
 
 class Storage(object):
-
-    def __init__(self, dirname, filename, config=None, cell_size=30,
+    """ Handles storage of shattered data in a TileDB structure. """
+    def __init__(self, dirname, filename, config= None, cell_size=30,
                  chunk_size=16):
         self.dirname = dirname
         self.src_file = filename
         self.config = config
         self.cell_size = cell_size
         self.chunk_size = chunk_size
+        ## TODO insert config here
+        self.ctx = tiledb.default_ctx()
 
+        self.atts = None
```

Comment:
Which atts are available is a property of the database. This should be a property method that pulls that info out of the tdb